facturacion_mexico/tests/test_setup_expense_item_groups.py

```
@@ -0,0 +1,105 @@
+			frappe.delete_doc("Item Group", child, force=True)
+	for parent in reversed(_ALL_PARENT_NAMES):
+		if frappe.db.exists("Item Group", parent):
+			frappe.delete_doc("Item Group", parent, force=True)
+	if frappe.db.exists("Item Group", _UMBRELLA):
+		frappe.delete_doc("Item Group", _UMBRELLA, force=True)
+	frappe.db.commit()
+
+
+class TestEnsureExpenseItemGroups(unittest.TestCase):
+	@classmethod
+	def setUpClass(cls):
```

Ruff (0.15.14)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)